composer/utils/checkpoint.py

```diff
@@ -1013,6 +1013,13 @@ def safe_torch_load(
                 'pass `load_ignore_keys = ["state/train_metrics/*", "state/eval_metrics/*"]`.',
             ) from e
         raise e
+    except FileNotFoundError as e:
+        if 'No such file or directory' in str(e) and dist.get_global_rank() != 0:
+            raise FileNotFoundError(
+                f'No such file or directory: {e.filename}. '
+                'Please check rank 0 for more downloading / debugging info.',
+            ) from e
```

Comment:
can we give a bit more description? eg check local rank 0 since its per node (and theres a dist fn to get the absolute rank iirc). I would also say something more explicit like:
No such file. This likely implies download failed on rank XX. Please check logs for that rank to see any information on how the download may have failed.